pkg/serverconfig/config.go

```diff
@@ -178,4 +220,21 @@ func addCustomization(fs *flag.FlagSet, customization *Customization) {
 	if customization.CustomLogoFile != "" {
 		fs.Set("custom-logo-file", customization.CustomLogoFile)
 	}
+
+	categories, err := json.Marshal(&customization.DeveloperCatalog.Categories)
```

Comment:
Since this is a pointer we should add check
```suggestion
	if &customization.DeveloperCatalog != nil && &customization.DeveloperCatalog.Categories != nil {
		categories, err := json.Marshal(&customization.DeveloperCatalog.Categories)
```

Reply:
Fixed, first check doesn't work. Second added as discussed in slack